samples/profiler/README.rst

```diff
@@ -0,0 +1,60 @@
+.. _profiler_sample:
+
+Profiler
+########
+
+The Profiler sample demonstrates the functionality of the Profiler subsystem.
+It shows how to use the Profiler to log and visualize data about custom events that are not part of the :ref:`event_manager`.
+
+Overview
+********
+
+The sample initializes the Profiler, registers two event types, and profiles their occurrences periodically.
+
+Event without data (``no data event``):
+  This event is used to signal the occurrence of an event only.
+  It does not contain additional data.
+
+Event with data (``data event``):
+  There are several numerical values associated with this event.
+  The values are updated periodically.
+
+
+Requirements
+************
+
+* One of the following development boards:
+
+  * nRF9160 DK board (PCA10090)
```

Comment:
does not match platform whitelist `nrf52_pca10040 nrf52840_pca10056`

Reply:
Ok, I will add it to whitelist